build.gradle

```diff
@@ -36,6 +36,7 @@ ext {
     kafka_avro_serializer_version = "6.1.0"
     avro_version = "1.10.1"
     webjars_locator_version = "0.40"
+    generatedMetamodels = "build/generated/sources/annotationProcessor/java/main/org/candlepin/subscriptions"
```

Comment:
Tell me more about this.  I assume it enables something cool!

Reply:
You're about to be disappointed - less cool than you'd think.

A little bit ago, I introduced Criteria, JPA Specifications, and Hibernate's Metamodel Generator to help facilitate that.  If you're using IntelliJ, the IDE is smart enough to just magically figure it out and resolve the dependencies when you have the editor open.  I started using VS Code again, which is no where near as smart as IntelliJ to figure those kind of things out, so the project/build file needs this for the VS Code gradle extension to get rid of all those "cannot resolve ______ class" warnings in the files dealing with the jpa Specifications.

Related, if VS Code didn't have a lombok extension, I'd need to add something similar to have VS Code recognize lombok annotations.  Eclipse has the same kind of issues.


TL;DR - I'm stubborn and can't give up my old IDE.

https://hibernate.org/orm/tooling/
https://docs.jboss.org/hibernate/orm/5.0/topical/html/metamodelgen/MetamodelGenerator.html